ui/src/redux/create.js

```diff
@@ -16,7 +16,7 @@ export default function createStore(llClient, router, data) {
   ];
 
   let finalCreateStore;
-  if (__DEVELOPMENT__ && __CLIENT__ && __DEVTOOLS__) {
+  if (__DEVELOPMENT__ && __CLIENT__ && (__DEVTOOLS__ || window.devToolsExtension)) {
```

Comment:
@ht2 can you check this line, I think you might have had an issue with this at some point.

Reply:
Yea - I believe the issue here is that when rendering server side (shareable dashboards), the window object does not exist, so you have to be careful. As far as I'm concerned the __DEVTOOLS_ should cover this, as long as your NODE_ENV=development, but that has other ramifications.

Parcel please https://media.giphy.com/media/nh9k1qzeLf99S/giphy.gif

Reply:
Yeah, remove the window reference check. 

Just add `ENABLE_REACT_DEVTOOLS=true` to env to enable

Reply:
Resolved in 1c4566f